vendor/types/@percy/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@percy';
```

Comment:
Is this file necessary?

Reply:
Yes. Without it there were `error TS2688: Cannot find type definition file for '@percy'. The file is in the program because: Entry point for implicit type library '@percy'` errors during e.g. `nx build playground`.

Reply:
I know why. Delete `vendor/types/@percy/index.d.ts` file, and then modify `tsconfig.base.json` (it was only looking for declaration files one level deep):
```
"typeRoots": ["./node_modules/@types", "./vendor/types/**/*.d.ts"],
```
Now the file in `vendor/types/@percy/sdk-utils/index.d.ts` should work.